tests/pilet.test.js

```diff
@@ -39,4 +41,43 @@ describe("pilet", () => {
         );
         expect(info.stderr).toBe("");
     });
+
+    it("pilet publish", async (done) => {
+        const pathToBuildDir = path.resolve(process.cwd(), "pilet");
+        const port = 9000;
+        const testFeedUrl = `http://localhost:${port}/api/v1/pilet`;
+        const testFeedKey = "df133a512569cbc85f69788d1b7ff5a909f6bcfe1c9a2794283a2fc35175882c";
+
+        const localFeedService = spawn(`npm run sample-feed`, {
+            cwd: process.cwd(),
+            shell: true
+        });
+
+        await sleep(5000); //wait for feed service to come up
```

Comment:
This does not seem to be reliable. Can we instead just poll e.g. `http://localhost:${port}/api/v1/pilet` until we get 200 back?

Reply:
I ran this multiple times and it always worked fine for me. I dont see the benefit of extra complexity here, but Nikita said he has a function for that in a similar case and wanted to take take of that.